### PR TITLE
MultiTenant Teams SSO

### PR DESCRIPTION
**Pull request by abdulzabbarshaik**
MultiTenant Teams SSO setup Steps

### CODE REVIEW

**Comment by PRMerger16** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 Image File | This PR contains an image file(s).

**Comment by PRMerger12** [comment]
@abdulzabbarshaik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md [hunk 1]
```diff
@@ -114,6 +114,47 @@ If you haven't, please follow the Installation steps first and then refer this a
 
     ![Save Application Settings.png](../../media/Save%20Application%20Settings%281%29.png)
 
+## Multi-Tenant support for Azure Active Directory based Authentication in Teams
+
+Microsoft Community Training platform supports login via multiple tenants in Teams for AAD based authentication instances.
+
+In this article, you will learn about how to setup Teams SSO login supporting multiple tenants to allow users of multiple tenants, use your MCT instance in Teams application.
```
Comment (v-sambati) [inline microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md:121]: Remove : to allow users of multiple tenants
```suggestion
In this article, you will learn about how to setup Teams SSO login supporting multiple tenants.
```

File: microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md [hunk 2]
```diff
@@ -114,6 +114,47 @@ If you haven't, please follow the Installation steps first and then refer this a
...
+
+### Steps to enable multi-tenant login in Teams for an AAD based instance
+
+1. Login to Azure portal
```
Comment (v-sambati) [inline microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md:125]: Link to azure
```suggestion
1. Login to [**Azure portal**](https://ms.portal.azure.com/#home)
```

File: microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md [hunk 3]
```diff
@@ -114,6 +114,47 @@ If you haven't, please follow the Installation steps first and then refer this a
...
+2. Go to App Services from the left-menu
+    :::image type="content" source="../../media/Teams SSO MultiTenant 1.png" alt-text="Teams SSO MultiTenant1":::
+3. Click on the app service belonging to your Microsoft Community Training instance.
+4. Select Configuration under settings from the left-menu
+    :::image type="content" source="../../media/Teams SSO MultiTenant 2.png" alt-text="Teams SSO 2":::
+5. Go under Applications Settings tab
+    :::image type="content" source="../../media/Teams SSO MultiTenant 3.png" alt-text="Teams SSO 3":::
+6. Search for `Features:IsMultiOrgDeployment`, add this if it does not exist, set the **value** as `true`, then click ‘Ok’
+7. Next, search for `idp:AzureADExternalAuthTenantId`, add this application setting if it does not exist using New application setting option
```
Comment (v-sambati) [inline microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md:134]:
```suggestion
7. Next, search for `idp:AzureADExternalAuthTenantId`, add this application setting if it does not exist using **New application setting** option
```

File: microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md [hunk 4]
```diff
@@ -114,6 +114,47 @@ If you haven't, please follow the Installation steps first and then refer this a
...
+8. Set the **value** as `common` replacing the existing tenant id (we suggest you keep a copy of your original Tenant ID value as a reference), then click ‘Ok’
+    :::image type="content" source="../../media/Teams SSO MultiTenant 4.png" alt-text="Teams SSO4":::
+9. Now while in Configurations section, search for `idp:AzureADExternalAuthTenant` and note the Tenant name
+10. Also, search for `idp:AzureADExternalAuthClientId` and note Client ID
+11. Navigate to your tenant (tenant name that you noted from Step 9) where your AAD exists, click on App registrations and search for application which corresponds to Client ID (Client ID that was noted from Step 10)
+    :::image type="content" source="../../media/Teams SSO MultiTenant 5.png" alt-text="Teams SSO 5":::
+12. Click on the application and navigate to ‘Authentication’ and select ‘Accounts in any organizational directory (Any Azure AD directory - Multitenant)’ under Supported account types and click ‘Save’
+    :::image type="content" source="../../media/Teams SSO MultiTenant 6.png" alt-text="Teams SSO 6":::
+13. For each tenant follow steps mentioned in the below two
```
Comment (v-sambati) [inline microsoft-community-training/infrastructure-management/install-your-platform-instance/create-teams-app-for-your-training-portal.md:143]:
```suggestion
13. For each tenant follow steps mentioned as below
```

**Review by v-sambati — CHANGES_REQUESTED** [review]

**Comment by PRMerger16** [comment]
@abdulzabbarshaik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@abdulzabbarshaik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 Image File | This PR contains an image file(s).

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 Image File | This PR contains an image file(s).

**Comment by PRMerger19** [comment]
@abdulzabbarshaik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@abdulzabbarshaik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 Image File | This PR contains an image file(s).

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 Image File | This PR contains an image file(s).

**Pull request merged**